### ISSUE TITLE
LASSO regression

### ISSUE THREAD

**Issue by VolodymyrOrlov** [smartcorelib/smartcore#29]
Implement LASSO Regression that is similar in functionality to [Scikit's Lasso](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Lasso.html).

We are looking to support following parameters:

* alpha
* normalize
* tol
* maxIter

[This paper](https://web.stanford.edu/~boyd/papers/pdf/l1_ls.pdf) describes an optimization method that is comparable to coordinate descent in solving large problems with modest accuracy, but is able to solve them with high accuracy with relatively small additional computational cost.

This method comes with a code that can be found [here](https://web.stanford.edu/~boyd/l1_ls/)
